README.md

```diff
@@ -113,49 +88,74 @@ through the Docker Compose
 docker-compose build
 ```
 
-### Start
+# How to start the request editor
 
 Use Docker Compose to run service locally.
 
 The service uses [Liquibase](https://www.liquibase.org/) to manage database migrations. To ensure the appropriate migrations have been run the utility script `scripts/start` may be run to execute the migrations, then the application.
 
-Alternatively the steps can be run manually:
+Alternatively to using `scripts/start` the steps can be run manually:
 * run migrations
   * `docker-compose -f docker-compose.migrate.yaml run --rm database-up`
 * start
   * `docker-compose up`
 * stop
   * `docker-compose down` or CTRL-C
 
+# How to get an output
+
+There are several different possible outputs for the request editor:
+
+1. **To access the web front-end for the service**  
+**Input:** Start the service (as described [above](#how-to-start-the-request-editor)).  
+**Output:** If you then go to `localhost:3001` (may be subject to change) you should see the home screen for the request editor's web front-end.  
+
+2. **To get a request awaiting debt data** (i.e. payment request for an amount less than the previously settled amount).  
+**Pre-requisite:** The customer in question already has a settled payment.  
+**Input:** Submit a payment request for the customer that is less than the customer's already settled balance.  
+**Output:** The request should now be awaiting debt data enrichment on the web front-end.  
+
+3. **To get a request awaiting ledger assignment or correction.**  
+**Pre-requisite:** Receive the output of 2. above.  
+**Input:** Using the web front-end, add debt enrichment data to the request.  
+**Output:** The request should now be awaiting ledger assignment or correction on the web front-end.  
 
+4. **To get a request awaiting ledger assignment quality check.**  
+**Pre-requisite:** Receive the output of 3. above.  
+**Input:** Using the web front-end sign off on the ledger assignment.  
+**Output:** The request should now be awaiting ledger quality check on the web front-end.  
+
+5. **To create a new debt dataset.**  
+**Input:** On the web front-end click on the Capture New Dataset button and fill in the details.  
+**Output:** The new dataset should now be added and viewable on the web front-end.  
+
+> Note: 3 and 4 are provisional measures requested by the RPA that may be removed at a future stage.
+
+Note that duplicate message detection is based on a `referenceId` property which must be a `UUID`.  If this property is not provided then the `invoiceNumber` property is used instead.
+
+When authentication is disabled, then the user will automatically be given all roles within the service and assigned a unique user Id.  For testing scenarios where multiple users are required, for example quality checks, then sign in again and a new Id will be assigned.
+
+
+# How to stop the request editor
+Run the command `docker-compose down`
```

Comment:
`docker-compose down -v` (again, removes data from db)

maybe you dnt want to remove the data from the db so we could have 2 ways of stopping it?